app.js

```diff
@@ -485,7 +485,9 @@ function checkForLookupRequests(output, callback) {
     } else if (data.context.action.lookup === 'branch') {
       console.log('************** Branch details *************** InputText : ' + payload.input.text);
       const loc = data.context.Location.toLowerCase();
-      bankingServices.getBranchInfo(loc, function(err, branchMaster) {
+      // Use the master (India) lookup for getBranchInfo
+      // It can return either India or US.
+      bankingServicesIN.getBranchInfo(loc, function(err, branchMaster) {
```

Comment:
`const bankingServicesIN = require('./banking_services');`
should this be renamed `banking_services_in`?

Reply:
Others LGTM

Reply:
I'll say no because we've been renaming/fixing things and it just breaks stuff.  Should the app be rewritten to be a better example?  Yes, but not until we have a test plan, script, etc... to make sure we don't add regressions.

Reply:
the variable name doesn't align with the file name. Since this is change for filename and it might break things, merging it for now. But we can revisit that later.